Non continous SVO's are preferably not serialized for their default values (#427)
Qowaiv/Qowaiv: README.md

* [Qowaiv.Json.Newtonsoft](https://www.nuget.org/packages/Qowaiv.Json.Newtonsoft/)  
* [Qowaiv.Text.Json.Serialization](https://www.nuget.org/packages/Qowaiv.Text.Json.Serialization/)  

> For the .NET 6.0, and higher versions of the package, when using `System.Text.Json`,

Reviewer: ```suggestion
For .NET 6.0, and higher versions of the package, when using `System.Text.Json`,
```

no custom serialization registration is required for Qowaiv SVO's: all have been
decorated with the `[JsonConverter]` attribute.

#### Do not serialize empty SVO's
Since .NET 8.0, it is possible to register modifiers for the `System.Text.Json.TypeInfoResolver`.

> This allows to change serialization behavior. On of the things to change is
> `ShouldSerialize(object model, object? prop)`. This can be useful if you do not
> want to serialize empty SVO's (such as `EmailAddress.Empty`). To get this
> (non-default) behavior you have to provide `JsonSerializerOptions` when serializing:

Reviewer: ```suggestion
This allows you to change serialization behavior, for example by editing
`ShouldSerialize(object model, object? prop)`. This can be useful if you do not
want to serialize empty SVO's (such as `EmailAddress.Empty`). To get this
(non-default) behavior you have to provide `JsonSerializerOptions` when serializing:
```
